client/controllers/searchInput.coffee

```diff
@@ -44,6 +45,15 @@ Template.searchInput.events
       instance.textFilter.set
         $regex: instance.$(event.target).val()
         $options: 'i'
+      if instance.data.tableId
+        Meteor.setTimeout(->
+          count = parseInt($("##{instance.data.tableId}").next().find('span.rows-per-page-count').text(), 10)
+          if count == 0
+            # hide loading indicator
+            $('.loading').hide()
+          else
+            $('.loading').show()
+        , 0)
```

Comment:
Maybe we could use `Meteor.defer` instead of `Meteor.setTimeout`? I suppose it would be a little cleaner.